instrument/D7_YIG_peaks.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" ?>
+<parameter-file date="2020-08-04" instrument="D7">
```

Comment:
Can you add comments to make clear that those are d-spacings of YIG in angstrom. Maybe put a reference to the paper.